src/hooks/index.ts

```diff
@@ -870,9 +906,19 @@ export const useResync = (wallet: YoroiWallet, options?: UseMutationOptions<void
   }
 }
 
-export const useNfts = (wallet: YoroiWallet) => {
+export const useNfts = (wallet: YoroiWallet, {search}: {search?: string} = {}) => {
   const utxos = useUtxos(wallet)
-  const query = useQuery({queryKey: [wallet.id, 'nfts', utxos.length], queryFn: () => wallet.fetchNfts()})
-  const {data: nfts = [], ...rest} = query
-  return {nfts, ...rest}
+  const {data, ...rest} = useQuery({
+    queryKey: [wallet.id, 'nfts', utxos.length],
```

Comment:
add a subscription to utxos, and invalidate the query

Reply:
Do you mean using the `useUtxos` hook?

Reply:
➕

Reply:
`useWallet`